Mathlib/Data/Set/Intervals/ProjIcc.lean

```diff
@@ -4,22 +4,30 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Yury G. Kudryashov, Patrick Massot
 
 ! This file was ported from Lean 3 source module data.set.intervals.proj_Icc
-! leanprover-community/mathlib commit 42e9a1fd3a99e10f82830349ba7f4f10e8961c2a
+! leanprover-community/mathlib commit 4e24c4bfcff371c71f7ba22050308aa17815626c
 ! Please do not edit these lines, except to modify the commit id
 ! if you have ported upstream changes.
 -/
 import Mathlib.Data.Set.Function
-import Mathlib.Data.Set.Intervals.Basic
+import Mathlib.Data.Set.Intervals.OrdConnected
 
 /-!
 # Projection of a line onto a closed interval
 
 Given a linearly ordered type `α`, in this file we define
 
+* `Set.projIci (a : α)` to be the map `α → [a, ∞[` sending `]-∞, a]` to `a`, and each point
```

Comment:
Don't we normally use `[a, ∞)` not `[a, ∞[`?

Reply:
Uh sure, but that's not what the mathlib docstring uses.